src/backend/api/meals.js

```diff
@@ -5,11 +5,119 @@ const knex = require("../database");
 router.get("/", async (request, response) => {
   try {
     // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    //const titles = await knex("meal_application").select("title");
+    //response.json(titles);
+    await knex("meals").insert({
```

Comment:
you could generate some meals by using mysql workbench and `database_dump_meals.sql` script